[2025-09-03T00:28:56.968+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: institution_detail_data_quality_pipeline.valid manual__2025-09-03T00:28:50.139230+00:00 [queued]>
[2025-09-03T00:28:56.974+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: institution_detail_data_quality_pipeline.valid manual__2025-09-03T00:28:50.139230+00:00 [queued]>
[2025-09-03T00:28:56.974+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T00:28:56.974+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-09-03T00:28:56.974+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T00:28:56.982+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): valid> on 2025-09-03 00:28:50.139230+00:00
[2025-09-03T00:28:56.985+0000] {standard_task_runner.py:55} INFO - Started process 17918 to run task
[2025-09-03T00:28:56.986+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'institution_detail_data_quality_pipeline', 'valid', 'manual__2025-09-03T00:28:50.139230+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/dag_dataquality.py', '--cfg-path', '/tmp/tmpjz1iggcb']
[2025-09-03T00:28:56.987+0000] {standard_task_runner.py:83} INFO - Job 108: Subtask valid
[2025-09-03T00:28:56.997+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-03T00:28:57.027+0000] {task_command.py:388} INFO - Running <TaskInstance: institution_detail_data_quality_pipeline.valid manual__2025-09-03T00:28:50.139230+00:00 [running]> on host f321ea8b8484
[2025-09-03T00:28:57.066+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Klaus_Rezende
AIRFLOW_CTX_DAG_ID=institution_detail_data_quality_pipeline
AIRFLOW_CTX_TASK_ID=valid
AIRFLOW_CTX_EXECUTION_DATE=2025-09-03T00:28:50.139230+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-09-03T00:28:50.139230+00:00
[2025-09-03T00:28:57.066+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-09-03T00:28:57.067+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo '✅ Data Quality validation passed'"]
[2025-09-03T00:28:57.073+0000] {subprocess.py:86} INFO - Output:
[2025-09-03T00:28:57.074+0000] {subprocess.py:93} INFO - ✅ Data Quality validation passed
[2025-09-03T00:28:57.074+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-03T00:28:57.092+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=institution_detail_data_quality_pipeline, task_id=valid, execution_date=20250903T002850, start_date=20250903T002856, end_date=20250903T002857
[2025-09-03T00:28:57.118+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-09-03T00:28:57.130+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
